docs/settings.py

```diff
@@ -1,10 +1,11 @@
 import os
+import logging
```

Comment:
```suggestion
```

Reply:
:ok_hand: All suggestions implemented in forced push.